src/app/agent/agent-dashboard/agent-dashboard.component.html

```diff
@@ -0,0 +1,89 @@
+<div id="agent-sidebar" class="active">
+  <ul *ngIf="user">
+    <li class="agent-data">Agent:<br>{{user.name}}</li>
+    <li class="agent-logout"><button type="button" class="btn btn-primary" (click)="logout()">Logout</button></li>
+    <li class="side-link" (click)="openBookFlight()">Book Flight</li>
+    <li class="side-link" (click)="openCancelFlight()">Cancel Flight</li>
```

Comment:
"Cancel Booking" or "Cancel Ticket" would probably be a better choice of words than "Cancel Flight".

Reply:
good call